rebar.config

```diff
@@ -40,5 +40,6 @@
 
 {plugins, [
     %% covertool,
-    {rebar3_eqc, {git, "https://github.com/Vagabond/rebar3-eqc-plugin", {branch, "master"}}}
+    {rebar3_eqc, {git, "https://github.com/Vagabond/rebar3-eqc-plugin", {branch, "master"}}},
+    rebar3_hex
```

Comment:
the hex plugin should be installed globally, not in the dep.

Reply:
ah yes, forgot to remove it from here, will do